plutarch-validators/WhalePoolsDex/Contracts/Proxy/FeeSwitch.hs

```diff
@@ -43,6 +44,7 @@ instance ToData DAOAction where
 data DAORedeemer = DAORedeemer
     { action    :: DAOAction
     , poolInIdx :: Integer
+    , poolNft   :: AssetClass
```

Comment:
In context of DAO validation there is no necessary to provide index directly, because we are expecting that tx created by our bot will place pool output at index 0, so the method that extract (and also verify that poolNft is exist in pool value) will do only one iteration